packages/app-next/src/App.tsx

```diff
@@ -85,12 +82,12 @@ const homePageExtension = createExtension({
 
 const app = createApp({
   features: [
-    graphiqlPlugin,
+    homePlugin,
     pagesPlugin,
+    graphiqlPlugin,
     techRadarPlugin,
-    techdocsPlugin,
     userSettingsPlugin,
-    homePlugin,
+    scmIntegrationsPlugin,
```

Comment:
I think we'll want to do what we're doing over in #20646 instead

Reply:
Done 🙌🏻